gulpfile.babel.js

```diff
@@ -65,7 +65,14 @@ gulp.task('lint:json', () => {
 
 gulp.task('format:json', () => {
   return gulp
-    .src(['./data-source/json/*.json'])
+    .src(['./data/*/*.json'])
```

Comment:
Formatting the data output directory, shouldn't be necessary - the source files should only need to be?